client/src/views/users/ActiveUsersPage.vue

```diff
@@ -189,13 +188,6 @@ const indeterminateState = computed({
   set: (_val) => _val,
 });
 
-const filteredUsers = computed(() => {
-  const revokedUsers = userList.value.filter((user) => {
-    return user.revoked === false;
-  });
-  return revokedUsers;
-});
```

Comment:
why removed filteredUsers from this page, but not from RevokedUsers?

Reply:
Here it can be filtered directly using the API. the function `listUsers` has a boolean parameter to include or not revoked users, so we don't need any filter later on.
In the case of the revoked user page, we don't have anything in the API to list only the revoked users, so we need to filter the list.